style.css

```diff
@@ -26,13 +24,27 @@ body {
   justify-self: flex-end;
 }
```

Comment:
- [ ] [OPTIONAL] Good job so far! Your project opens only the same size in the mobile version, resulting in a white field on the right when expanded from 300 px to 800 px. My advice to you would be to ask you to fix this problem. Let's learn [additional information](https://youtu.be/yU7jJ3NbPdA).

![image](https://user-images.githubusercontent.com/63915024/170840774-8c62eb86-9e0d-4218-9f17-6cd80284114d.png)